start.php

```diff
@@ -170,7 +172,10 @@
 		fs_update_sdk_newest_version( $this_sdk_relative_path, $fs_active_plugins->plugins[ $this_sdk_relative_path ]->plugin_path );
 
 		$is_current_sdk_newest = true;
-	} else if ( version_compare( $fs_active_plugins->newest->version, $this_sdk_version, '<' ) ) {
+
+        // Try to reorder plugins so that the plugin containing the newest SDK version will be loaded first.
+        $redirect_after_reordering_plugins = ( true === fs_newest_sdk_plugin_first() );
```

Comment:
@fajardoleo I see the signature and code of the function `fs_newest_sdk_plugin_first()`. The return type is `bool`. So is this `( true === fs_newest_sdk_plugin_first() )` comparison necessary?

```php
$redirect_after_reordering_plugins = fs_newest_sdk_plugin_first()
```